ddtrace/llmobs/_utils.py

```diff
@@ -205,6 +207,18 @@ def _unserializable_default_repr(obj):
         return "[Unserializable object: {}]".format(repr(obj))
 
 
+def _get_base_url() -> str:
+    subdomain = ""
+    site = DEFAULT_DD_SITE
```

Comment:
I don't think we need this constant, `config._dd_site` is `datadoghq.com` by default

Reply:
Yeah we probably don't need this check below, since the default site is datadoghq.com anyway. A user would need to really set the wrong site for it to mess up

Reply:
sounds good